contact.php

```diff
@@ -1,33 +1,73 @@
+<?php
+ini_set('display_errors', 1);
+require_once('config.php');
+//Check for form submission - Is the form empty? 
+if (!empty($_POST)) {
+	//Connect and escape
+	//$mysql = new mysqli(DB_HOST, DB_USER, DB_PASS, DB_NAME);
+	//Submit to database
+	//$query = "INSERT INTO users (name,email) VALUES ('$name','$email')";
+	//$insert = $mysql->query($query);
+}
+?>
 <!DOCTYPE html>
 <html lang="en">
-    
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"/>
-    
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous" />
 </head>
 
 <body>
-    <div name="form">
-        <form method="post" name="your-message">
-            Full Name: <input type="text" name="fullName"/><br>
-            Email: <input type="email" name="email"/><br>
-            Your Message: <textarea name="yourMessage">type here</textarea><br>
-            <input type="submit" value="send message"/>
-        </form>
-    </div>
+	<div class="container">
+		<div name="form">
+			<?php
+			//Define variables set to empty values
+			$name = '';
+			$email = '';
+			$nameErr = '';
+			$emailErr = '';
 
-    <?php 
-        $fullName = $_POST["fullName"];
-        $email = $_POST["email"];
-        $yourMessage = $_POST["yourMessage"];
-
-        echo "Hello $fullName <br>";
-        echo "We will respond to your message @ $email <br>";
-        
-    ?>
-    
+			if (!empty($_POST['submit'])) {
+				var_dump($_POST);
+				//Check if name was submitted
+				if (empty($_POST['name'])) {
+					$nameErr = "Name is required.";
+				} else {
+					$name = $_POST['name'];
+				}
+				//Check if email is submitted
+				if (empty($_POST['email'])) {
+					$emailErr = "Email is required";
+				} else {
+					$email = $_POST['email'];
+				}
+			}
+			?>
+			<form action="" method="post">
+				<div class="form-title-row">
+					<h1>Sign up for our movie pass newsletter!</h1>
+				</div>
+				<div class="form-field">
+					<label for="name">Your Name</label>
+					<input type="text" name="name" placeholder="Your Name Here" value="<?php if (isset($_POST['name'])) echo htmlspecialchars($_POST['name']); ?>">
```

Comment:
reduce this if to either a ternary or [null coalece](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)

Reply:
Did we get rid of the htmlspecialchars?